openedx/core/djangoapps/django_comment_common/comment_client/course.py

```diff
@@ -29,17 +30,8 @@ def get_course_commentable_counts(course_key: CourseKey) -> Dict[str, Dict[str,
             }
 
     """
-    url = f"{settings.PREFIX}/commentables/{course_key}/counts"
-    response = perform_request(
-        'get',
-        url,
-        metric_tags=[
-            f"course_key:{course_key}",
-            "function:get_course_commentable_counts",
-        ],
-        metric_action='commentable_stats.retrieve',
-    )
-    return response
+    commentable_stats = forum_api.retrieve_commentables_stats(str(course_key))
```

Comment:
We don't see the "retrieve" verb very often in APIs. Let's use "get" instead, i.e: "get_commentable_stats".